ajax/libs/tempusdominus-bootstrap-4/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tempusdominus-bootstrap-4",
+  "author": "Jonathan Peterson",
+  "description": "A date/time picker component designed to work with Bootstrap 4 and Momentjs. For usage, installation and demos see Project Site on GitHub",
+  "homepage": "https://tempusdominus.github.io/bootstrap-4/",
+  "license": "MIT",
+  "keywords": [
+    "bootstrap",
+    "datepicker",
+    "datetimepicker",
+    "timepicker",
+    "moment"
+  ],
+  "filename": "js/tempusdominus-bootstrap-4.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tempusdominus/bootstrap-4.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tempusdominus/bootstrap-4.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "*/"
```

Comment:
Let's use `**/*`.
The current pattern can't get the files from git repo.
As extend1994 [said](https://github.com/cdnjs/cdnjs/pull/12337#discussion_r157361881) 
(You can fix this part via GitHub web-site, we can help you squash the commit.)
Thanks.